c2corg_ui/static/js/documentediting.js

```diff
@@ -374,6 +394,23 @@ app.DocumentEditingController.prototype.handleMapFeatureChange_ = function(
 
 
 /**
+ * It doesn't associate, it just pushes it into associations array.
+ * @private
+ */
+app.DocumentEditingController.prototype.pushDocToAssociations_ = function() {
+  var doctype = this.urlParams_['doctype'];
+  var docid = this.urlParams_['docToAssociateId'];
+  $.get(this.apiUrl_ + '/' + doctype + '/' + docid, function(doc) {
```

Comment:
I mean if you make a request to `/search?q={document_id}&t=r` instead of `/routes/{document_id}` you will not get the full document but only the information you need.